sessionize/app/src/main/java/co/touchlab/sessionize/schedule/EventAdapter.kt

```diff
@@ -56,50 +45,49 @@ class EventAdapter(private val context: Context,
         updateData()
     }
 
-    inner class ScheduleBlockViewHolder(itemView: View) : RecyclerView.ViewHolder(itemView), EventRow {
-
-        override fun setTitleText(s: String) {
-            itemView.findViewById<TextView>(R.id.title).text = s
-        }
-
-        override fun setTimeText(s: String) {
-            itemView.findViewById<TextView>(R.id.time).text = s
-        }
-
-        override fun setSpeakerText(s: String) {
-            itemView.findViewById<TextView>(R.id.speaker).text = s
-        }
-
-        override fun setDescription(s: String) {
-            // Field only exists for tablet
-            itemView.findViewById<TextView>(R.id.event_description)?.text = s
-        }
-
-        override fun setLiveNowVisible(liveNow: Boolean) {
-            itemView.findViewById<ImageView>(R.id.live)
-                    .visibility = if (liveNow) View.VISIBLE else View.INVISIBLE
+    inner class ScheduleBlockViewHolder(itemView: View) : RecyclerView.ViewHolder(itemView) {
+
+        private val title = itemView.findViewById<TextView>(R.id.title)
```

Comment:
This finds the necessary views on construction, rather than every time the view gets recycled